File: Tesseract.Database/Models/Employee.cs
using System;
using System.Collections.Generic;
using System.Text;
using Tesseract.Database.Models.Base;

namespace Tesseract.Database.Models
{
    public class Employee : Person
    {
        public virtual IEnumerable<Dependent> Dependents { get; set; }
    }
}

File: Tesseract.Client/src/app/models/Employee.ts
import { Person } from "./base/Person";
import { Dependent } from "./Dependent";
import { ResourcesService } from "../resources/resources.services";

export class Employee extends Person {
    dependents: Dependent[] = [];

    constructor(resourcesService: ResourcesService) {
        super(resourcesService)
    }
}